components/Frame/DarkmodeSwitch.js

```diff
@@ -67,18 +71,20 @@ const DarkmodeSwitch = ({ colorSchemeKey: pageColorSchemeKey, t }) => {
             </Radio>
             <br />
             {/*
-              // ToDo activating auto 
-              // - uncomment Radio below
+              // ToDo App has been updated 
+              // - remove condition below
             */}
-            {/*<Radio
-              checked={!colorSchemeKey}
-              onChange={() => {
-                // default is undefined
-                setColorSchemeKey()
-              }}
-            >
-              {t('darkmode/switch/auto')}
-            </Radio>*/}
+            {!inNativeApp ? (
+              <Radio
+                checked={!colorSchemeKey}
+                onChange={() => {
+                  // default is undefined
+                  setColorSchemeKey()
```

Comment:
```suggestion
                  setColorSchemeKey('auto')
```

Reply:
shouldn't it still be default white, with just the option to set it as auto?

Reply:
Well here we need to explicitly save `auto` into the local storage.

When there is no local storage—or a empty value—the frontend is configured to activate light (via color synch & head script)

Reply:
ah logo...